eiffel-vocabulary/EiffelAnnouncementAcknowledgedEvent.md

```diff
@@ -19,7 +19,36 @@
 The EiffelServiceDeployedEvent is to declare that an EiffelAnnouncementPublishedEvent has been received & acknowledged by a person or a machine.
 
 ## Data Members
- 
+### data.acknowledgement
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The Acknowledgement response.
+
+### data.user
+__Type:__ String  
+__Required:__ No  
```

Comment:
Do the combination of requiring a Description but not who/what has acknowledged make sense?

Reply:
We are trying to be generic as possible, giving the user and their UC options whether they want to make themselves known or not.
What is important is that we capture the fact the "something" has acknowledged the Announcement and publishes that fact.